pom.xml

```diff
@@ -85,6 +86,38 @@
             <scope>test</scope>
         </dependency>
 	</dependencies>
+
```

Comment:
@artur-ciocanu this is quite good. but we probably need to follow the Dapr Java SDK formatting stuff..

Reply:
I have used the Dapr Java SDK checkstyle.xml.